provisioning/roles/mariadb/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+- set_fact:
+    database_type: mysql
+
+- name: Add MariaDB apt key
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: 1BB943DB
+  become: yes
+
+- name: Add MariaDB repo
+  apt_repository:
+    repo: "deb http://mirror.mva-n.net/mariadb/repo/{{ mariadb_version }}/debian wheezy main"
```

Comment:
`wheezy` is not the right value for most case here. It should be determined based on the base boy. Maybe @OdyX  can help if you don't know how to do that.

FYI, it is done in the `php` task also to select the correct repository.

Reply:
Also, as a general comment, `mariadb` is available in both Debian and Ubuntu directly. We user the official repo only because we needed a version not available on our Debian version.

Proposing to chose a version is complex. Both the `php` and `mysql` roles are monsters because of it.